server.js

```diff
@@ -2,14 +2,14 @@
 
 const express = require('express');
 const PORT = 3000;
-const app = express();
+const exp = express();
```

Comment:
Server code is completely separate from client-side code, so this change should not be necessary.